doc/changelogs/CHANGELOG_V18.md

```diff
@@ -174,7 +242,7 @@ Certificates removed:
 * corepack updated to 0.25.2.
 * ICU updated to 74.2. Includes CLDR 44.1 and Unicode 15.1.
 * npm updated to 10.5.0. Fixes a regression in signals not being passed onto child processes.
-* simdutf8 updated to 4.0.8.
+* simdutf updated to 4.0.8.
```

Comment:
Is it OK to amend changelog of previous versions?

Reply:
I've undone this change.